### PR TITLE
Move Manage User to Required options; secure root in bare-metal-install-server

### PR DESCRIPTION
**Pull request by mvincerx**
- Align with clr-installer merged PR: https://github.com/clearlinux/clr-installer/pull/576
- Replace Figures 18-21 to reflect TUI with Admin selected by default
- Renumber all figures after 18 and reassign

Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/get-started/bare-metal-install-server.rst [hunk 1]
```diff
@@ -468,6 +468,100 @@ be configured post-installation using the ``cryptsetup`` tool.
...
+#. Enter a :guilabel:`Password`.
+
+   .. note:
+
+      Minimum length is 8 characters. Maximum length is 255 characters.
+
+#. In :guilabel:`Confirm`, enter the same password.
+
+#. The :guilabel:`Administrator` checkbox is selected by default.
+   This assures a secure boot immediately after installation.
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-server.rst]:
```suggestion
 .. note::	

      Selecting Administrator enables sudo privileges for the user. At least one user must be assigned as an Administrator for the installation to proceed.
```

Comment (mdhorn) [reply source/get-started/bare-metal-install-server.rst]: I was also trying to eliminate this wording you had on line 514 "This assures a secure boot immediately after installation."  "Secure boot" has a specific meaning that might be misinterpreted.
If you want some text there, we should rephrase that something like "Defining an administrator user account allows the secure locking of the `root` account." or just drop the line which is what I had intended.

**Comment by mvincerx** [comment]
I apologize. I intended to remove that before. That line is removed. See latest commit.

**Comment by mvincerx** [comment]
Easy-p. So, do you want this published now, or hold until next week?

**Comment by mdhorn** [comment]
We might hold this until next week when the installer releases.
Unless you want to get it taken care of now, and it will just reflect how the install will work.

**Comment by mvincerx** [comment]
Please add @ callout to me when you release next week. We'll keep docs aligned with your release.

**Comment by mvincerx** [comment]
@mdhorn. Please let me know when. ;)

**Comment by mdhorn** [comment]
@mvincerx Code has been tagged, and it will be releated to Koji withing the hour.
Should be available in tonights build, and visible in the images once released late tomorrow barring any QA finds.

**Comment by mvincerx** [comment]
Thanks. I'll hold until final QA review. Let me know when you do, please.

**Comment by mdhorn** [comment]
@mvincerx Clear Linux OS 31380 was released today and contains the update version of clr-installer. This change can now be published.

**Comment by mvincerx** [comment]
Cheers.  Thanks, @mdhorn.

**Pull request merged**